protostar/starknet_gateway/block_explorer/viewblock_block_explorer.py

```diff
@@ -17,10 +17,8 @@ def get_name(self) -> str:
     def create_link_to_transaction(self, tx_hash: TransactionHash) -> Optional[str]:
         return f"{self._prefix}/tx/0x{tx_hash:064x}{self._get_network_query_param()}"
 
-    def create_link_to_contract(
-        self, contract_address: ContractAddress
-    ) -> Optional[str]:
-        return f"{self._prefix}/contract/0x{contract_address:064x}{self._get_network_query_param()}"
+    def create_link_to_contract(self, contract_address: Address) -> Optional[str]:
+        return f"{self._prefix}/contract/0x{int(contract_address):064x}{self._get_network_query_param()}"
```

Comment:
regular string interpolation will do the trick here

Reply:
resolving in favor of https://github.com/software-mansion/protostar/pull/1170#discussion_r1025363409